src/secure_random.cr

```diff
@@ -1,6 +1,16 @@
+require "base64"
 require "openssl/lib_crypto"
 
 module SecureRandom
+  def self.base64(n = 16)
+    Base64.encode64(random_bytes(n)).delete('\n')
+  end
+
+  def self.urlsafe_base64(n = 16, padding = false)
+    str = base64(n).tr("+/", "_")
+    padding ? str : str.delete('=')
+  end
+
```

Comment:
Intents are differents: Base64 encodes data when SecureRandom returns a token of a given size. I agree those 2 methods are merely sugar for `encode64(random_bytes(n))`.

But you're right: there is a urlsafe_encode64 that I should be using!